benchmarks/visualize.py

```
@@ -361,7 +361,7 @@ def series(data):
         return {size: t for size, t in pts}
 
     e = series(eager)
-    l = series(lazy)
+    l = series(view)
```

Ruff (0.15.20)

[error] Ambiguous variable name: `l`

(E741)